frontend/src/screens/Home.tsx

```diff
@@ -59,7 +60,15 @@ function Home() {
 
   return (
     <>
-      <AppHeader title={getGreeting(albyMe?.name)} description="" />
+      <AppHeader
```

Comment:
When the user has Alby Pro would it be nice to show somewhere on the app that they have Alby Pro and "thank you for supporting Alby"?

Reply:
Yeah, I like that idea. I'd add it with the new sidebar though as I think a main menu item is a bit too prominent and I'd rather move that to some dropdown: (maybe instead of "Upgrade"?

![image](https://github.com/user-attachments/assets/71f3d427-d645-48c6-9af7-15a58065fc71)